### ISSUE TITLE
Regarding Japanese ASR

### ISSUE THREAD

**Issue by tejassp2002** [espnet/espnet#4467]
Hi @sw005320! 
Thanks for this amazing library! 

Basically, I'm dealing with Japanese ASR for longer speech. The existing pre-trained Japanese Model in model_zoo only works for audio<5sec. 
1. I thought of checking out the streaming ASR for that instance, but unfortunately I couldn't find any pre-trained model of the [Streaming Transformer](https://arxiv.org/pdf/2006.14941.pdf) ASR for Japanese (though in the paper authors have trained on CSJ). Could you please upload the model if possible. 
2. I also wanted to try [Chunking with stride](https://huggingface.co/blog/asr-chunking#chunking-with-stride) but I'm not sure how can I get the logits out of the pre-trained ASR model. [Speech2Text](https://espnet.github.io/espnet/_modules/espnet2/bin/asr_inference_maskctc.html#Speech2Text) doesn't have any feature to return the logits. Could you please help me out with this! 

Thanks.

**Comment by sw005320** [comment]
Thanks for your report!

> 1. I thought of checking out the streaming ASR for that instance, but unfortunately I couldn't find any pre-trained model of the [Streaming Transformer](https://arxiv.org/pdf/2006.14941.pdf) ASR for Japanese (though in the paper authors have trained on CSJ). Could you please upload the model if possible.

@eml914, could I ask you whether you can upload some of your Japanese ASR models?
If not, we’ll prepare them.

> 2. I also wanted to try [Chunking with stride](https://huggingface.co/blog/asr-chunking#chunking-with-stride) but I'm not sure how can I get the logits out of the pre-trained ASR model. [Speech2Text](https://espnet.github.io/espnet/_modules/espnet2/bin/asr_inference_maskctc.html#Speech2Text) doesn't have any feature to return the logits. Could you please help me out with this!

The logits output is not supported in the Speech2Text interface.
You need to make a new Speech2Text interface for this purpose.

We also usually recommend some VAD tools (e.g., Kaldi or webrtc vad) to segment the audio in advance.

**Comment by tejassp2002** [comment]
Thanks @sw005320 for your quick response! 

I would really appreciate if @eml914 could upload some models for Japanese streaming ASR. Currently, I have no model from ESPnet for Japanese ASR which works for longer audios for benchmarking. 

Also, thanks for the VAD suggestion.

**Comment by hcchiu1202** [comment]
@sw005320 @eml914 Thank you for all of your fantastic contribution. May I ask if you still have the plan to prepare some pre-trained Japanese streaming ASR models?

**Comment by eml914** [comment]
Hi @hcchiu1202 , thank you for your question.
I now have some Japanese models only under peculiar conditions, so I am now preparing a general streaming Japanese model for sharing.  Thank you for your patient and I will let you know when I upload the model.
